statsmodels/tsa/tests/test_stattools.py

```diff
@@ -243,6 +243,46 @@ def test_granger_fails_on_nobs_check(self):
         assert_raises(ValueError, grangercausalitytests, X, 3, verbose=False)
 
 
+class SetupKPSS(object):
+    data = macrodata.load()
+    x = data.data['realgdp']
+
+
+class TestKPSS(SetupKPSS):
+    def test_fail_nonvector_input(self):
+        kpss(self.x)  # should be fine
+
+        x = np.random.rand(20, 2)
+        assert_raises(ValueError, kpss, x)
+
+    def test_fail_unclear_hypothesis(self):
+        hypo = "level"
+        kpss(self.x, hypo)  # should be fine
+
+        hypo = "trend"
+        kpss(self.x, hypo)  # should be fine also
+        assert_raises(ValueError, kpss, self.x, "unclear hypothesis")
+
+    def test_teststat(self):
+        kpss_stat, pval, lags, flag, crits = kpss(self.x, "level", 3)
+        assert_almost_equal(kpss_stat, 5.0169, DECIMAL_3)
```

Comment:
Can you put in the class-level docstring (or a separate file if it's long) some indication of the R code (I assume it was R-code) used to generate these test values?